Sources/SwiftInspectorVisitors/ProtocolVisitor.swift

```diff
@@ -56,16 +56,18 @@ public final class ProtocolVisitor: SyntaxVisitor {
     let typealiasVisitor = TypealiasVisitor(parentType: .simple(name: name))
     typealiasVisitor.walk(node.members)
 
+    let propertiesVisitor = PropertySyntaxVisitor()
```

Comment:
non-blocking: thoughts on renaming `PropertySyntaxVisitor` -> `PropertyVisitor` for consistency?

Reply:
Good call 👍

Reply:
https://github.com/fdiaz/SwiftInspector/pull/100/commits/b45d12bb8e8e178896a95599b66977bdc21bd3df